src/axom/quest/examples/quest_signed_distance_interface.cpp

```diff
@@ -52,21 +55,74 @@ int numranks;
 /*!
  * \brief Holds command-line arguments
  */
-static struct
+struct Arguments
 {
   std::string fileName;
-  int ndims;
-  int maxLevels;
-  int maxOccupancy;
-  int box_dims[3];
-  double box_min[3];
-  double box_max[3];
-  bool specified_box_min;
-  bool specified_box_max;
-  bool is_water_tight;
-  bool dump_vtk;
-  bool use_shared;
-} Arguments;
+  int ndims{3};
+  int maxLevels{15};
+  int maxOccupancy{5};
+  std::vector<int> box_dims{32,32,32};
```

Comment:
We're not actually using that type in the code -- it is explicitly cast to an `axom::IndexType` below when it is passed into the `mint` mesh.

I think it is better to keep this as an `int` in case the user supplies a negative number, it is more likely to get caught below.

Reply:
Nevermind -- `IndexType` is a signed int -- I'll make the change.